code/datums/brain_damage/mild.dm

```diff
@@ -0,0 +1,240 @@
+
+/datum/brain_trauma/mild/concussion/on_life()
+	if(prob(25))
+		switch(rand(1,11))
+			if(1)
+				to_chat(owner, "<span class='notice'>Your stomach writhes with pain.</span>")
+				owner.vomit()
+			if(2,3)
+				to_chat(owner, "<span class='notice'>You feel light-headed.</span>")
+				owner.dizziness += 10
+			if(4,5)
+				to_chat(owner, "<span class='notice'>it becomes hard to see for some reason.</span>")
+				owner.confused += 10
+				owner.apply_effect(10,EYE_BLUR)
+			if(6 to 9)
+				to_chat(owner, "<span class='notice'>Your tongue feels thick in your mouth.</span>")
+				owner.slurring += 30
+			if(10)
+				to_chat(owner, "<span class='notice'>You forget for a moment what you were doing.</span>")
+				owner.Stun(20)
+			if(11)
+				to_chat(owner, "<span class='warning'>You faint.</span>")
+				owner.Sleeping(80)
+
+	..()
+
+/datum/brain_trauma/mild/muscle_weakness
+	name = "Muscle Weakness"
+	desc = "Patient experiences occasional bouts of muscle weakness."
+	scan_desc = "weak motor nerve signal"
+	gain_text = "<span class='warning'>Your muscles feel oddly faint.</span>"
+	lose_text = "<span class='notice'>You feel in control of your muscles again.</span>"
+
+/datum/brain_trauma/mild/muscle_weakness/on_life()
+	var/fall_chance = 5
+	if(owner.m_intent == "run")
+		fall_chance += 15
+	if(prob(fall_chance) && !owner.lying && !owner.buckled)
+		to_chat(owner, "<span class='warning'>Your leg gives out!</span>")
+		owner.Weaken(5)
+
+	else if(owner.get_active_hand())
+		var/drop_chance = 15
+		var/obj/item/I = owner.get_active_hand()
+		drop_chance += I.w_class
+		if(prob(drop_chance) && owner.drop_from_inventory(I))
+			to_chat(owner, "<span class='warning'>You drop [I]!</span>")
+
+	else if(prob(3))
+		to_chat(owner, "<span class='warning'>You feel a sudden weakness in your muscles!</span>")
+		owner.adjustHalLoss(25)
+	..()
+
+/datum/brain_trauma/mild/muscle_spasms
+	name = "Muscle Spasms"
+	desc = "Patient has occasional muscle spasms, causing them to move unintentionally."
+	scan_desc = "nervous fits"
+	gain_text = "<span class='warning'>Your muscles feel oddly faint.</span>"
+	lose_text = "<span class='notice'>You feel in control of your muscles again.</span>"
+
+/datum/brain_trauma/mild/muscle_spasms/on_life()
+	if(prob(25))
+		switch(rand(1,5))
+			if(1)
+				if(owner.canmove)
+					to_chat(owner, "<span class='warning'>Your leg spasms!</span>")
+					step(owner, pick(cardinal))
+			if(2)
+				if(owner.incapacitated())
+					return
+				var/obj/item/I = owner.get_active_hand()
+				if(I)
+					to_chat(owner, "<span class='warning'>Your fingers spasm!</span>")
+					I.attack_self(owner)
+			if(3)
+				var/prev_intent = owner.a_intent
+				owner.a_intent = I_HURT
+
+				var/range = 1
+				if(istype(owner.get_active_hand(), /obj/item/weapon/gun)) //get targets to shoot at
+					range = 7
+
+				var/list/mob/living/targets = list()
+				for(var/mob/M in oview(owner, range))
```

Comment:
Randomly aiming at and shooting mobs seems a bit silly. Maybe it should just pick a random (simulated and visible) atom or turf near the player?

Reply:
Noted.